openmpi-2.1.1-disable-lsf-support.patch

```diff
@@ -0,0 +1,29 @@
+--- a/config/orte_check_lsf.m4	2017-05-10 17:40:48.000000000 +0200
```

Comment:
I got the patch from https://github.com/open-mpi/ompi/issues/3546
openmpi.spec has the tarball version for openmpi included
the patch is has the version number in the name
is this the revision information you had in mind?